extensions/yafw/src/utils.tsx

```diff
@@ -109,3 +109,7 @@ export function capitalizeSnakeCase(text: string): string {
     .map((part) => part.charAt(0).toUpperCase() + part.slice(1))
     .join(" ");
 }
+
+export function isFileFormatSupported(filename: string): boolean {
+  return VIDEO_FORMATS.some((format) => filename.toLowerCase().endsWith(format));
+}
```

Comment:
```suggestion
export function isFileFormatSupported(_filename: string): boolean {
  const filename = _filename.toLowerCase()
  return VIDEO_FORMATS.some((format) => filename.endsWith(format));
}
```

so we don't repeat "toLowerCase()" once per extension

Reply:
good catch thanks :) 

I applied your suggestion